File: 351004/Lazuta/src/entities/Label.cpp
#include "Label.h"

Label::Label() : Entity()
{
    m_name = "";
}

Label::Label(const std::string& name)
    : Entity()
{
    m_name = name;
}

Label::~Label()
{
}

std::string Label::getName() const
{
    return m_name;
}

void Label::setName(const std::string& name)
{
    m_name = name;
}

File: 351001/polozkov/src/main/java/org/polozkov/entity/label/Label.java
package org.polozkov.entity.label;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;
import org.polozkov.entity.issue.Issue;

import java.util.List;

@Entity
@Getter
@Setter
@Table(name = "tbl_label")
public class Label {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String name;

    @ManyToMany(mappedBy = "labels", fetch = FetchType.LAZY) // Указываем поле из класса Issue
    private List<Issue> issues;
}
